src/components/HomeBlocks/LeaderboardsHomeBlock.tsx

```diff
@@ -30,30 +32,44 @@ export const LeaderboardsHomeBlock = ({ homeBlock }: Props) => {
   const { leaderboards } = homeBlock;
   const metadata = homeBlock.metadata as HomeBlockMetaSchema;
 
-  console.log(homeBlock.leaderboards);
-
   return (
     <HomeBlockWrapper className={classes.root}>
       {metadata?.title && (
-        <Group position="apart" align="center" pb="md">
-          <Title>{metadata?.title}</Title>
+        <Group position="apart" align="center" pb="md" style={{ flexWrap: 'nowrap' }}>
+          <Title
+            sx={(theme) => ({
+              fontSize: isMobile
+                ? theme.headings.sizes.h3.fontSize
+                : theme.headings.sizes.h1.fontSize,
+            })}
```

Comment:
I'd rather prefer using css mediaQueries over js to set the font size on different screen sizes